WordPress/src/main/java/org/wordpress/android/ui/WPTooltipView.kt

```diff
@@ -0,0 +1,92 @@
+package org.wordpress.android.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.ImageView
+import android.widget.LinearLayout
+import android.widget.TextView
+import androidx.annotation.LayoutRes
+import org.wordpress.android.R
+import org.wordpress.android.ui.WPTooltipView.TooltipPosition.ABOVE
+import org.wordpress.android.ui.WPTooltipView.TooltipPosition.BELOW
+import org.wordpress.android.ui.WPTooltipView.TooltipPosition.LEFT
+import org.wordpress.android.ui.WPTooltipView.TooltipPosition.RIGHT
+import org.wordpress.android.util.RtlUtils
+import java.lang.IllegalArgumentException
+
+/**
+ * Partially based on https://stackoverflow.com/a/42756576
+ *
+ * NOTE: this is a very basic implementation of a tooltip component
+ * mainly used to cover the need of onboarding/announcing in the IA Project main create FAB.
+ * More work/rework is needed to make it a full custom view tooltip component.
+ * A different and more dynamic approach that can be used is with PopupWindow taking care of behaviors like
+ * CoordinatorLayout behavior (as in this scenario with FAB and snackbars)
+ */
+
+class WPTooltipView @JvmOverloads constructor (
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : LinearLayout(context, attrs, defStyleAttr) {
+    var position = LEFT
+    var messageId = 0
+
+    init {
+        attrs?.also {
+            val a = context.theme.obtainStyledAttributes(
```

Comment:
maybe give this field a bit better name? for example `stylesAttributes`?

Reply:
Done here d5bb53872c114f53a917d58ef346a4406db1132e